extensions/amp-base-carousel/1.0/arrow.js

```diff
@@ -25,44 +25,36 @@ import {dict} from '../../../src/utils/object';
 export function Arrow(props) {
   const {
     'customArrow': customArrow,
-    'dir': dir,
+    'by': by,
     'advance': advance,
     'disabled': disabled,
   } = props;
-  const button = customArrow
-    ? customArrow
-    : renderDefaultArrow(dict({'dir': dir}));
   const {
-    'children': children,
     'disabled': customDisabled,
     'onClick': onCustomClick,
-    ...rest
-  } = button.props;
+  } = customArrow.props;
   const isDisabled = disabled || customDisabled;
   const onClick = (e) => {
     if (onCustomClick) {
       onCustomClick(e);
     }
-    advance();
+    advance(by);
   };
   return (
     <div
       style={{
         ...styles.arrowPlacement,
         // Offset button from the edge.
-        [dir < 0 ? 'left' : 'right']: '0px',
+        [by < 0 ? 'left' : 'right']: '0px',
+        opacity: isDisabled && 0,
+        pointerEvents: isDisabled && 'none',
```

Comment:
Is `pointerEvents: false` a right value in the `style` map? And the same q for `opacity` above.

Reply:
Good catch, updated this.